### ISSUE TITLE
Fools phaseblade of quickness pickline question about maxdamage per level

### ISSUE THREAD

**Issue by loldermond** [kolton/d2bot-with-kolbot#2300]
i wrote this to find a "fools phaseblade of quickness" with 2 sockets. how do i put the maxdamage per level attribute of it?

https://i.imgur.com/2vxfoVh.png

[Name] == PhaseBlade && [Quality] == Magic # [ias] >= 40 && ([itemtohitperlevel] > 0 || [itemtohitpercentperlevel] > 0) && [sockets] >= 2

**Comment by chriseng440** [comment]
[Name] == phaseblade && quality == magic # [ias] == 40 && [itemtohitpercentperlevel] >= 0 && [ItemMaxDamagePerlevel >= 0 && [sockets] == 2

**Comment by david1602** [comment]
@chriseng440 missing the `]` after `ItemMaxDamagePerLevel`, and I think it should be `>0` on both, not `>=0`, though I'm not sure on the latter

**Comment by chriseng440** [comment]
yeah [itemmaxdamageperlevel]here one of my pickit that works[Name] == DimensionalBlade && [Quality] == Rare && [Flag] == Ethereal # [EnhancedDamage] >= 200 && [ItemMaxDamagePerLevel] >= 1 && [ItemToHitPerLevel] >= 1 

    On Monday, February 24, 2020, 06:01:52 PM EST, David Dernovoj <notifications@github.com> wrote:  
 
 
@chriseng440 missing the ] after ItemMaxDamagePerLevel, and I think it should be >0 on both, not >=0, though I'm not sure on the latter
